src/views/preview/project-view.jsx

```diff
@@ -947,6 +950,9 @@ const mapDispatchToProps = dispatch => ({
         dispatch(previewActions.resetComments());
         dispatch(previewActions.getTopLevelComments(id, 0, isAdmin, token));
     },
+    handleReduxProjectInfo: info => {
```

Comment:
I think this could be much simpler if we just connected this in `FormsyProjectUpdater`.